presto-mixin/dashboards/presto-coordinator-overview.libsonnet

```diff
@@ -0,0 +1,1343 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'presto-coordinator-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nonheapMemoryUsagePanel = {
```

Comment:
For all the gauges can you please verify the thresholds are set appropriately? Currently they're all using the panel defaults for yellow and red at 70% and 80% respectively

Reply:
Yea these should be good. Generally from what i've seen, anywhere from 0-70% could be considered safe on heap/non-heap, with 80-90% considered warning levels, and above being bad!